From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: David Stephens <dave@force9.org>
Date: Sat, 24 Oct 2026 16:14:58 +0100
Subject: [PATCH 2/2] Say this is a modified PRoot built for Starboard

--version and --help name Starboard and link to the source.
---
 src/cli/proot.h | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/src/cli/proot.h b/src/cli/proot.h
index 3f61931..32444e6 100644
--- a/src/cli/proot.h
+++ b/src/cli/proot.h
@@ -1,5 +1,8 @@
 /* This file is automatically generated from the documentation. EDIT AT YOUR OWN RISK. */
 
+/* Modified by the Starboard project (https://get-starboard.app),
+ * 2026-10-24: the subtitle and colophon say this is a modified PRoot.  */
+
 #ifndef PROOT_CLI_H
 #define PROOT_CLI_H
 
@@ -74,9 +77,12 @@ static int post_initialize_exe(Tracee *, const Cli *, size_t, char *const *, siz
 static Cli proot_cli = {
 	.version  = VERSION,
 	.name     = "proot",
-	.subtitle = "chroot, mount --bind, and binfmt_misc without privilege/setup",
+	.subtitle = "chroot, mount --bind, and binfmt_misc without privilege/setup (modified for Starboard)",
 	.synopsis = "proot [option] ... [command]",
-	.colophon = "Visit http://proot.me for help, bug reports, suggestions, patchs, ...\n\
+	.colophon = "This is a modified version of PRoot, built for Starboard: https://get-starboard.app\n\
+Source and changes: https://github.com/get-starboard/starboard-third-party\n\
+\n\
+Visit http://proot.me for help, bug reports, suggestions, patchs, ...\n\
 Copyright (C) 2015 STMicroelectronics, licensed under GPL v2 or later.",
 	.logo = "\
  _____ _____              ___\n\
